api-server/src/roster/models.py

```diff
@@ -387,3 +390,126 @@ class UserImage(models.Model):
+                    },
+                    "angle": {
+                        "type": "integer",
+                        "description": "this is the angle to which the text will be rotated. Default is 0"
+                    },
+                    "img_size": {
+                        "type": "object",
+                        "properties": {
+                            "width": {
+                                "type": "integer",
+                                "description": "Width of the image that will contain the text"
+                            },
+                            "height": {
+                                "type": "integer",
+                                "description": "Height of the image that will contain the text"
+                            }
+                        },
+                        "required": [
+                            "width",
+                            "height"
+                        ]
+                    },
+                    "font_size": {
+                        "type": "integer",
+                        "description": "Font size of the custom text"
+                    },
+                    "font_opacity": {
+                        "type": "number",
+                        "minimum": 0,
+                        "maximum": 1,
+                        "description": "Opacity of the custom text. Must be between 0 and 1"
+                    },
+                    "text_offset_x": {
+                        "type": "integer",
+                        "description": "Spacing of custom text within its image, X axis"
+                    },
+                    "text_offset_y": {
+                        "type": "integer",
+                        "description": "Spacing of custom text within its image, Y axis"
+                    },
+                    "font_alignment": {
+                        "type": "string",
+                        "description": "Alignment of custom text. I.e., center, left, right"
+                    }
+                },
+                "required": [
+                    "x",
+                    "y",
+                    "prop",
+                    "img_size",
+                    "text_offset_x",
+                    "text_offset_y"
+                ]
+            }
+        ]
+    }
+    name = models.CharField(max_length=128)
+    dds_file = models.FileField(upload_to='livery_dds', unique=True)
+    json_description = models.JSONField(
+        blank=True,
+        null=True,
+        validators=[LiverySkinJsonValidator(limit_value=JSON_FIELD_SCHEMA)]
+    )
+
+    def __str__(self):
+        return self.name
+
+
+class Livery(models.Model):
+
+    squadron = models.ForeignKey(Squadron, on_delete=models.CASCADE)
+    positions = ['co', 'xo', 'opso', '']
+    position_helper = {i + 1: x for i, x in enumerate(positions)}
```

Comment:
```suggestion
    positions = Aviator.positions
    position_helper = Aviator.postiion_helper
```

Just to keep it DRY and at the moment i can't think of a case where if we were to update it on the aviator we'd also not want to update it here.

Reply:
made changes as suggested